src/main/resources/db/changelog/20240802101300-update-anon-cert-table-cert-column.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet id="20240802101300-1" author="jalbrech">
+        <modifyDataType tableName="cp_anonymous_certificates"
+            columnName="cert"
+            newDataType="LONGBLOB" />
```

Comment:
now you can include the datatypes.xml file at the top, and then use the `${cert.type}` variable instead of hardcoding to LONGBLOB here

Reply:
I ended up using an SQL statement to explicitly define the blob length to avoid alias translations based on the comment from Chris:

https://github.com/candlepin/candlepin/pull/4799#discussion_r1701981622

Reply:
This won't work on PostgreSQL: The alter syntax is wrong, and blob isn't a known type.

For PostgreSQL, the correct syntax for the type is:
`ALTER TABLE cp_anonymous_certificates ALTER COLUMN cert SET DATA TYPE <type>;`

Reply:
I think Nikos has the right idea here where we keep using the modifyDataType op, but pull in the "new" definitions from datatypes.xml for the change. If we need special types for each DB, we can define them accordingly.

Reply:
Updated to use the `${cert.type}` variable